internal/cmd/common.go

```diff
@@ -26,6 +26,10 @@ func readMarkdownFile(args []string) ([]byte, error) {
 	if arg == "" {
 		f, err := os.DirFS(fChdir).Open(fFileName)
 		if err != nil {
+			if pathError := (*os.PathError)(nil); errors.As(err, &pathError) {
```

Comment:
That's clever :) Idiomatic go would be with `var err *os.PathError` in a separate line.